packages/icons/src/index.story.js

```diff
@@ -9,22 +9,37 @@ function Container({ children }) {
 
 const allStories = storiesOf("Icons", module);
 
-const noStoryComponents = [
-  "ClassicLegIcon",
-  "ClassicModeIcon",
-  "LegIcon",
-  "StandardLegIcon",
-  "StandardModeIcon",
-  "TriMetLegIcon",
-  "TriMetModeIcon"
-];
+const bikeRentalItinerary = require("./__mocks__/bike-rental-gbfs.json");
 
-Object.keys(Icons).forEach(iconKey => {
-  if (noStoryComponents.indexOf(iconKey) > -1) return;
-  const Component = Icons[iconKey];
-  allStories.add(iconKey, () => (
-    <Container>
-      <Component />
-    </Container>
-  ));
+const [walkLeg, bikeRentalLeg] = bikeRentalItinerary.legs;
```

Comment:
It seems the added code for testing the rendering of rental/hail legs should be in its own `.story.js` file and create a new story heading such as `storiesOf("Icons in bike rental legs", module);` or something similar.

Reply:
Yea, you're probably right.